src/main/msp/msp.c

```diff
@@ -3294,8 +3295,13 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, int16_t cmdMSP,
 #else
         sbufReadU8(src);
 #endif
-        break;
 
+#ifdef USE_RANGEFINDER
+        rangefinderConfigMutable()->rangefinder_hardware = sbufReadU8(src);
```

Comment:
`MSP_GET_SENSOR_CONFIG` should match SET
Maybe delay MSP_SET_SENSOR_CONFIG for follow up PR. CLI is enough for testing